lib/ex_unit/test/ex_unit/doc_test_test.exs

```diff
@@ -458,6 +529,7 @@ defmodule ExUnit.DocTestTest do
       defmodule NeverCompiled do
         import ExUnit.DocTest
         doctest ExUnit.DocTestTest.IndentationMismatchedPrompt
+        doctest ExUnit.DocTestTest.FencedMismatchedPrompt
```

Comment:
We won't reach this line because of the failure on previous line.
This should be moved to its own module. Same applies to all other tests below.